src/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSVN.java

```diff
@@ -255,18 +273,14 @@ private boolean findSourceCodeSVN(HttpMessage originalMessage) throws Exception
 			//svnsourcefileattackmsg.setRequestHeader(this.getBaseMsg().getRequestHeader());
 			sendAndReceive(svnsourcefileattackmsg, false);  //do not follow redirects
 
```

Comment:
This is the biggest part of this change, in three places (here and two below) we check attack strength and if LOW or MED ignore all 400 and 500 series responses (return false). Otherwise, as long as it's not part of the unwanted list (established earlier) we continue.